packages/Tag/src/Tag.tsx

```diff
@@ -19,9 +19,10 @@ export type AppearanceNew =
   | "positive"
   | "caution"
   | "negative"
-  | "neutral";
+  | "neutral"
+  | "select";
```

Comment:
whats a "select" appearance? looks like a dropdown? is there new figma files related to this?

Reply:
Well I was trying to figure out the best way to handle the select tag:
![image](https://github.com/gsoft-inc/ov-igloo-ui/assets/110498164/f23861d0-7509-47d0-a7bd-99c33f6396f9)

Before we over-rid it in the tag picker but this just didn't make sense so I tried to create a new appearance. I'm not sure if there was a better way to do this. It seemed weird to change the styles based on if it had a dismiss button or not.
I know that maybe it should have been a new component, but at this point since the dismiss button and icons were already in the Tag component, I didn't think it made sense for Igloo.

Any better ideas?

Reply:
In Hopper, this should be 2 components like this: 
https://wl-orbiter-website.netlify.app/?path=/docs/lozenge--default-story
https://wl-orbiter-website.netlify.app/?path=/docs/tag--default-story#default

For now we'll add a variant in igloo if it's the wayyou want to proceed